fdbserver/storageserver.actor.cpp

```diff
@@ -3826,6 +3836,7 @@ ACTOR Future<Void> getKeyValuesQ(StorageServer* data, GetKeyValuesRequest req)
 		} else {
 			state int remainingLimitBytes = req.limitBytes;
 
+			state double kvReadRange = g_network->timer();
```

Comment:
The latency is always recorded, do we want to add a knob to control the sample frequency?

Reply:
When we create the latency, we give latency logging internal which handles the frequency.

Example:
kvReadRangeLatencySample("KVGetRangeMetrics",
 		                             self->thisServerID,
 		                             SERVER_KNOBS->LATENCY_METRICS_LOGGING_INTERVAL,
 		                             SERVER_KNOBS->LATENCY_SAMPLE_SIZE),